schema/Bridge.graphql

```diff
@@ -128,6 +129,10 @@ type CrossTransfer @entity {
   """
   votes: Int # uint8
   """
+  IsSigned - is this transfer signed by the federation contract (only relevant for the new federation contracts)
```

Comment:
How does this relate to the status property? Does Signed mean that the tx is executed? If so, can we just use the status property instead of adding a new property?

Reply:
signed means this specific transfer has been voted by all 3 federators and soon after it will executed (signed and executed are both fired in the same tx)
this only happens in the new federation contract
so this is basically an indicator that this transfer was executed with the new federation contract

Reply:
this is meant to signal how this transfer got approved, by the "legacy" system or by the new system